### PR TITLE
Initial use of pandoc-ebook-template.

### PR DESCRIPTION
**Pull request by Rho-n**
This uses the suggested template. It has room for improvement, but does look nice.

### CODE REVIEW

**Comment by Rho-n** [comment]
If you decide to use the template, I am happy to help work with the project to tweak the output settings or set up a different LaTex template used for the conversion to pdf.  I will also help set up the make script to deal with converting translations.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+BUILD = build
+BOOKNAME = monero-gui-wallet-guide
+TITLE = title.txt
+METADATA = metadata.xml
+CHAPTERS = ch00.md ch01.md ch02.md ch03.md ch04.md ch05.md ch06.md ch08.md ch09.md
+TOC = --toc --toc-depth=2
+COVER_IMAGE = media/monero-logo-1280.png
```
Comment (erciccione) [inline Makefile]: I see you didn't upload a logo. Is this just a placeholder?

File: metadata.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<dc:rights>Copyright 2013, erciccione</dc:rights>
```
Comment (erciccione) [inline metadata.xml]: This guide is not copyrighted or licensed, so this string can be left empy for now. I plan to add a license soon btw

File: Makefile (hunk 1 above)
Comment (Rho-n) [reply Makefile]: Whoops, no I forgot to commit it.

File: metadata.xml (hunk 2 above)
Comment (Rho-n) [reply metadata.xml]: Okay, I wasn't sure and put it in as a place holder and example.

File: Makefile (hunk 1 above)
Comment (Rho-n) [reply Makefile]: > I see you used the documentclass 'book'. I tested it using 'report' and it was looking much better IMO

Which ever template looks best to you is fine with me. My report template must be slightly different than yours...when I tried to use it, I got an error about \frontmatter not being defined.

**Comment by pazos** [comment]
the html output can be placed at http://monero-ecosystem.github.io/ using github pages and the gui just open that page on a external browser. I would like the idea more than https://github.com/monero-project/monero-gui/pull/1602.

**Comment by Rho-n** [comment]
I agree with removing the numbering from the heading. There should be a way to apply it through the conversion template--if desired.

File: Makefile (hunk 1 above)
Comment (erciccione) [reply Makefile]: I think `/frontmatter` and ` \mainmatter ` are not recognized by the *report* template. I removed them during my tests, and it worked just fine

**Comment by erciccione** [comment]
>    use toc-depth=3
    let pandoc generate our toc based on h1-h3 headers (and remove numbers from headers)

@Rho-n Could you implement those changes in this PR?

Also, could you set up the makefile to deal with translations? after that we can merge this PR and keep working on the template with other pull requests

> the html output can be placed at http://monero-ecosystem.github.io/ using github pages and the gui just open that page on a external browser

I like this Idea, I have to open an issue on meta about this, since i cannot use `http://monero-ecosystem.github.io/` only for the guide. That could be used to show all projects inside the ecosystem, but we could create a section an put the guide in there. Don't know if this could substitute monero-project/monero-gui#1602. I prefer to give an offline solution, since, thanks to this template, we can easily provide 3 different formats of this guide locally. Shouldn't be a problem. Or we could just link from the GUI to the local HTML file

**Comment by Rho-n** [comment]
I have an initial system set up to handle the translations. I tried to keep  it a simple as possible. Currently there is one master Makefile in the root of the repository, and each translation will be in its own directory. I was thinking the standard name for a translation directory would be its two letter language code, but it can anything you want.

A LANGUAGES variable is set at the top of the root Makefile. It must contain the list of translation directories. A sub Makefile must be in every translation directory. The one in the en directory can be copied over to any new translation folders without changes.

To create various formats and languages

make all  (creates all formats for all languages)
make epub (creates epub for all languages)
make html (creates html for all languages)
make pdf (creates pdf for all languages)
make <format>-<translation folder> (make a particular format for the particular language only) 
make html-en (make html for english translation)

There is room for improvement, but I think this is a good start.

**Pull request merged**